commit 986779949b46d4dff533005b9987394643d710fb
Author: popcornmix <popcornmix@gmail.com>
Date:   Mon Jul 17 16:54:06 2017 +0100

    Revert "bcm2835-mmc: Fix DMA usage"
    
    This reverts commit f4258b9352afc1480dd1c29a11528e53b73bf356.

diff --git a/drivers/mmc/host/bcm2835-mmc.c b/drivers/mmc/host/bcm2835-mmc.c
index 981db05de1ff..4fe8d1fe4457 100644
--- a/drivers/mmc/host/bcm2835-mmc.c
+++ b/drivers/mmc/host/bcm2835-mmc.c
@@ -1354,14 +1354,14 @@ static int bcm2835_mmc_add_host(struct bcm2835_host *host)
 		if (ret == 0) {
 			host->dma_cfg_rx = cfg;
 
-			host->have_dma = true;
+			host->use_dma = true;
 		} else {
 			pr_err("%s: unable to configure DMA channel. "
-			       "Falling back to PIO\n",
+			       "Faling back to PIO\n",
 			       mmc_hostname(mmc));
 			dma_release_channel(host->dma_chan_rxtx);
 			host->dma_chan_rxtx = NULL;
-			host->have_dma = false;
+			host->use_dma = false;
 		}
 	}
 #endif
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1232_986779949b46d4dff533005b9987394643d710fb.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1232_986779949b46d4dff533005b9987394643d710fb.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
